ZelloChannelKit/ZelloChannelKit/network/SocketRocket/Delegate/ZCCSRDelegateController.m

```diff
@@ -47,7 +47,7 @@ - (instancetype)init
 - (void)setDelegate:(id<ZCCSRWebSocketDelegate> _Nullable)delegate
 {
     dispatch_barrier_async(self.accessQueue, ^{
-        _delegate = delegate;
+        self->_delegate = delegate;
```

Comment:
I don't think I remember ever seeing `this->` in ObjC...

Reply:
Facebook updated the SocketRocket library last year for the first time since like 2016. It may have fixed the issues I made changes for here, but updating the SDK code to use an imported SocketRocket library might be out of scope for these changes.